src/js/components/TextInput/doc.js

```diff
@@ -71,7 +71,13 @@ export const doc = TextInput => {
     ),
     onSelect: PropTypes.func.description(
       `Function that will be called when the user selects a suggestion.
-The suggestion contains the object chosen from the supplied suggestions.`,
+      The suggestion contains the object chosen from the supplied suggestions.
```

Comment:
I would add at the beginning of the description that this function is deprecated and the user should use the onSuggestionSelect instead.